grpc/rust/lib.rs

```diff
@@ -15,4 +15,4 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 //
 
-tonic::include_proto!("typedb.protocol");
+include!("typedb.protocol.rs");
```

Comment:
The old form `tonic::include_proto!` is basically designed for Cargo build scripts; it depends on an environment variable (`OUT_DIR`) that they set. The `include!` macro allows us to easily reference generated code provided we know its exact location.